client/v3/watch.go

```diff
@@ -717,9 +716,9 @@ func (w *watchGrpcStream) run() {
 	}
 }
 
-func shouldRetryWatch(cancelReasonError error) bool {
-	return (strings.Compare(cancelReasonError.Error(), v3rpc.ErrGRPCInvalidAuthToken.Error()) == 0) ||
-		(strings.Compare(cancelReasonError.Error(), v3rpc.ErrGRPCAuthOldRevision.Error()) == 0)
+func shouldRetryWatch(cancelReason string) bool {
+	return (strings.Compare(cancelReason, v3rpc.ErrGRPCInvalidAuthToken.Error()) == 0) ||
```

Comment:
can we use `==` instead of strings.Compare since we only care about equality and not ordering

Reply:
Resolved & thx